OrbitGl/EventTrack.cpp

```diff
@@ -187,15 +187,53 @@ bool EventTrack::IsEmpty() const {
 }
 
 //-----------------------------------------------------------------------------
-std::string EventTrack::GetSampleTooltip(PickingID id) const {
-  auto SafeGetFormattedFunctionName = [](uint64_t addr) -> std::string {
-    auto it = Capture::GAddressToFunctionName.find(addr);
-    return it == Capture::GAddressToFunctionName.end()
-               ? std::string("<i>") + "???" + "</i>"
-               : it->second;
-  };
+std::string SafeGetFormattedFunctionName(uint64_t addr, int max_line_length) {
```

Comment:
This function is not declared in any header file, is it? In that case it should be static (or in an anonymous namespace).

Reply:
Done